.github/workflows/llvm-project-tests.yml

```diff
@@ -104,8 +104,15 @@ jobs:
           PKG_CONFIG_PATH: /usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig//12
         shell: bash
         run: |
+          if [ "${{ runner.os }}" == "Linux" ]; then
+            builddir="/mnt/build/"
+            sudo mkdir -p $builddir
```

Comment:
Why are the additional permissions needed here?

Reply:
The /mnt directory is owned by root:
https://github.com/tstellar/llvm-project/actions/runs/7772011497/job/21193943451

Reply:
Interesting. Approach seems fine to me then.

Would be really nice if Github wouldn't use this weird layout for their hosted runners, but I guess it is what it is.